xboxdash/main.c

```diff
@@ -1,21 +1,42 @@
-#include <xboxrt/debug.h>
-#include <pbkit/pbkit.h>
+#include <winapi/windows.h>
+#include <hal/debug.h>
+#include <hal/video.h>
 #include <hal/xbox.h>
-#include "stdio.h"
+#include <xboxkrnl/xboxkrnl.h>
+#include <nxdk/mount.h>
 
-void main(void)
+#define TrayStateHasMedia 0x60
+
+int main(void)
 {
-    int i;
+    XVideoSetMode(640, 480, 16, REFRESH_DEFAULT);
 
-    if (pb_init()) {
-        XSleep(2000);
-        XReboot();
-        return;
+    if (!nxMountDrive('A', "\\Device\\CdRom0\\")) {
```

Comment:
Not sure if mounting the drive is necessary - I'd consider running `XLaunchXBE()` with the full path instead.